lms/djangoapps/class_dashboard/dashboard_data.py

```diff
@@ -263,8 +290,13 @@ def get_d3_sequential_open_distrib(course_id):
             subsection_name = own_metadata(subsection).get('display_name', '')
 
             num_students = 0
-            if subsection.location in sequential_open_distrib:
-                num_students = sequential_open_distrib[subsection.location]
+
+            try:
+                module = client.modules(course_id, subsection.location)
+                sequential_open = module.sequential_open_distribution()
+                num_students = sequential_open[0]['count']
```

Comment:
That is correct.  Strictly speaking, the sequential open distribution API does not need to return a list right now, since to the best of my knowledge there is only one count maintained per module_id, so the list is always of length one.  That said, the other module level API endpoints (grade and answer distributions) also return lists, because they track multiple values.  I think it makes sense to maintain consistency between their return types and allow for the possibility that we track opening multiple parts of a module in the future, but I could be wrong.